client/catapult/scripts/ci/test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+echo "tbd"
```

Comment:
tbd? 😁

Reply:
yea, we eventually will want to run cpp unit tests ^^